roles/openshift_hosted/tasks/registry.yml

```diff
@@ -43,7 +43,7 @@
 
 - name: Update registry environment variables when pushing via dns
   set_fact:
-    openshift_hosted_registry_env_vars: "{{ openshift_hosted_registry_env_vars | combine({'OPENSHIFT_DEFAULT_REGISTRY':'docker-registry.default.svc:5000'}) }}"
+    openshift_hosted_registry_env_vars: "{{ openshift_hosted_registry_env_vars | combine({'REGISTRY_OPENSHIFT_SERVER_ADDR':'docker-registry.default.svc:5000'}) }}"
```

Comment:
Is it possible to keep the variable with the naming scheme that we use?
How about?
`OPENSHIFT_REGISTRY_SERVER_ADDR`

Reply:
Sorry, this variable name is dictated by some docker distribution config conventions, we can't change it (also I really wouldn't want to introduce yet another supported variable name... we've already shipped support for 3 different variable names for this config setting (so we have 2 deprecated ones)).